charts/galoy/templates/admin-deployment.yaml

```diff
@@ -18,11 +18,11 @@ metadata:
 spec:
 
   replicas: {{ .Values.galoy.admin.replicas }}
-  
+
```

Comment:
could we run a yaml formatter to avoid those diff?

Reply:
We can have a github action like we have for terraform formatting. That would fail if there were empty spaces being committed, like there were in #1389.